tests/features/tokens/generate.feature

```diff
@@ -0,0 +1,87 @@
+
+
+   Scenario Outline: Admin cannot generate Token using HTTP API with invalid expires
+        Given the admin has the admin password
+        And the admin gets an 'access_token' by logging in to '/api/v1/token' with a 'write:token' scope
+        And the admin adds Authorization Bearer 'access_token' in the 'headers'
+        And the admin adds JSON payload with scopes: <scopes> and expires: <expires>
+        When the admin sends a POST request to '/api/v1/token/new' with invalid 'expires' in 'payload'
+        Then the admin should get status code '422'
+
+        Examples:
+            | scopes                             | expires |
+            | ['read:bootstrap','write:targets'] | 0       |
+            | ['read:bootstrap','write:targets'] | -5      |
+            | ['read:bootstrap']                 | None    |
+
+
+   Scenario Outline: Admin cannot generate Token using HTTP API for certain scopes
+        Given the admin has the admin password
+        And the admin gets an 'access_token' by logging in to '/api/v1/token' with a 'write:token' scope
+        And the admin adds Authorization Bearer 'access_token' in the 'headers'
+        And the admin adds JSON payload with scopes: <scopes> and expires: <expires>
+        When the admin sends a POST request to '/api/v1/token/new' with not allowed 'scopes' in 'payload'
+        Then the admin should get status code '422'
+
+        Examples:
+            | scopes                               | expires |
+            | ['write:token']                      | 24      |
+            | ['read:bootstrap','write:token']     | 24      |
+            | ['write:bootstrap']                  | 24      |
+            | ['write:bootstrap', 'read:settings'] | 24      |
+            | []                                   | 24      |
+            | ['']                                 | 24      |
+
+   Scenario Outline: Admin is Unauthorized to generate using HTTP API with invalid token
+        Given the admin adds Authorization Bearer <token> in the 'headers'
+        And the admin adds JSON payload with scopes: <scopes> and expires: <expires>
+        When the admin sends a POST request to '/api/v1/token/new' with invalid 'access_token' in the headers
+        Then the admin should get status code '401'
+        And the admin should get 'Failed to validate token' in body
+
+        Examples:
+            | token       | scopes             | expires |
+            | invalid     | ['write:targets']  | 1       |
+            | eyJhbiJIUzI | ['read:bootstrap'] | 1       |
+            | ''          | ['read:bootstrap'] | 1       |
+
+    Scenario Outline: Admin uses RSTUF Command Line Interface to generate Token
+        Given the admin has repository-service-tuf (rstuf) installed
+        And the admin is logged in using RSTUF Command Line Interface
+        And the admin types 'rstuf -c rstuf.ini admin token generate -s <scopes_params> -e <expires_params>"
```

Comment:
Can you double-check if the right quotes are used here, i.e. is it correct to use `"` instead of ``\`` at the end?